### PR TITLE
Commiting Search AutoComplete

### PR DESCRIPTION
**Pull request by sudeepcb**
Added AutoComplete items: autocomplete.js, search.ejs
Added initial test files: connection.js

### CODE REVIEW

File: webApp/client/style.css [hunk 1]
```diff
@@ -38,7 +38,7 @@
 .body {
     height: 100%;
     background-color: black;
-    overflow: auto;
+    overflow: hidden;
```
Comment (tim-nguyen-cs) [inline webApp/client/style.css:41]: Might have to go over

File: webApp/client/style.css [hunk 2]
```diff
@@ -54,7 +54,7 @@
     background: linear-gradient(rgba(255, 255, 255, .5), rgba(255, 255, 255, .5)), url("background.png");
     height: 150%;
     background-position: center center;
-    background-repeat: no-repeat;
+    background-repeat: repeat;
```
Comment (tim-nguyen-cs) [inline webApp/client/style.css:57]: Also will go over

File: webApp/client/style.css [hunk 3]
```diff
@@ -124,7 +124,6 @@
 .search-bar {
     position: relative;
     text-align: center;
-    top: -30px;
```
Comment (tim-nguyen-cs) [inline webApp/client/style.css:127]: Under review

**Review by tim-nguyen-cs — APPROVED** [review]
Changes necessary to implement typeAhead feature

**Review by noll115 — APPROVED** [review]
It all checks out

**Pull request merged**